src/daemon/rpc_command_executor.cpp

```diff
@@ -723,35 +725,39 @@ bool t_rpc_command_executor::print_transaction(crypto::hash transaction_hash) {
     {
       // only available for new style answers
       if (res.txs.front().in_pool)
-        tools::success_msg_writer() << "Found in pool";
+        tools::success_msg_writer() << "Found in pool" << std::endl;
```

Comment:
The message writers handle the newline already.

Reply:
This is to add an additional blank line, so that there is a consistent blank line after whatever is written (with json and hex or not) when the command is executed. Ok to remove if you prefer; but then would remove blank line in other cases too for consistency. I think ideally it should be consistent across all commands and possible outputs to finish with a blank line (or not).

Reply:
I'd rather not start changing all this pretty printing now.

Reply:
ok